lib/ddtrace/contrib/extensions.rb

```diff
@@ -86,6 +88,15 @@ def fetch_integration(name)
             registry[name] ||
               raise(InvalidIntegrationError, "'#{name}' is not a valid integration.")
           end
+
+          def integration_disabled?(name)
+            env_to_bool(
+              "#{ENV_INTEGRATION_DISABLED_PREFIX}_"\
+              "#{name.to_s.sub(':', '').upcase}"\
```

Comment:
will this always match the same name used in `DD_TRACE_<NAME>_ANALYTICS_ENABLED` that you set above?

Reply:
yup, it's the key being passed into `c.use` which are all either one word or snake case. I think this is just a convention though and not enforced anywhere afaik, so should probably add a test to prevent a regression at some point.